modules/clock-analogue.xml

```diff
@@ -207,6 +207,10 @@ if (typeof updateClock === 'function') {
     setInterval(updateClock, 1000);
 }
     ]]></onVisible>
+    <onRender><![CDATA[
+      // Scale the content
+      $(target).xiboLayoutScaler(properties);
```

Comment:
Should this be `globalOptions` instead of `properties` ?

Reply:
The `onRender` function receives `properties` as a parameter. Which includes `globalOptions` as defined in `player.js`
![image](https://github.com/xibosignage/xibo-cms/assets/128448242/2de1b812-4907-4572-840d-e96207c0a7a4)

Should we proceed by just `globalOptions` OR make use of `properties`?

Reply:
I think we should use the same variable as in the other files, and in player.js we are indeed merging `widget.properties` and `globalOptions`.